### PR TITLE
nodes that contain id within logic app should be skipped

### PR DESCRIPTION
**Pull request by bobbyangers**

### CODE REVIEW

File: arm-ttk/testcases/deploymentTemplate/IDs-Should-Be-Derived-From-ResourceIDs.test.ps1 [hunk 1]
```diff
@@ -61,8 +61,10 @@ foreach ($id in $ids) { # Then loop over each object with an ID
         }
     }
 
-    
-    
+    # Skip this check if the property is within a logic app
+    if ( $id.ParentObject.type -imatch '^microsoft.logic/workflows$' ) {
```
Comment (bmoore-msft) [inline arm-ttk/testcases/deploymentTemplate/IDs-Should-Be-Derived-From-ResourceIDs.test.ps1:65]: I think the . may have some side effects in the regex but practically speaking if that was your type property the template has other problems.  I'm ok with this as is - @StartAutomating can you have a quick look?

**Comment by bobbyangers** [comment]
I tried to do amend.... reopening

File: arm-ttk/testcases/deploymentTemplate/IDs-Should-Be-Derived-From-ResourceIDs.test.ps1 (hunk 1 above)
Comment (bobbyangers) [reply arm-ttk/testcases/deploymentTemplate/IDs-Should-Be-Derived-From-ResourceIDs.test.ps1:65]: Can you please reopen ?

**Comment by bmoore-msft** [comment]
I can't reopen - probably because the branch is gone...  But you can just submit another.

**Pull request closed without merging**